dashboard/src/components/views/EnvironmentView/EventsContent.js

```diff
@@ -11,7 +11,7 @@ import Query from "/components/util/Query";
 import { withQueryParams } from "/components/QueryParams";
 
 // If none given default expression is used.
-const defaultExpression = "HasCheck";
```

Comment:
You can use the `has_check` property here.

Reply:
Ah, I see now @ `SynthesizeExtras` 👍